_integration_tests/mise/install_java_test.go

```diff
@@ -29,20 +29,27 @@ func TestMiseInstallJavaVersion(t *testing.T) {
 		},
 		{
 			name:               "Temurin major version only",
-			requestedVersion:   "temurin-21",
+			requestedVersion:   "temurin-22",
```

Comment:
Use EOL versions in tests so that we won't get broken tests when a new version is released.